ext/curl/curl.stub.php

```diff
@@ -15,9 +17,21 @@ function curl_errno($handle): int {}
 /** @param resource $handle */
 function curl_error($handle): string {}
 
+#if LIBCURL_VERSION_NUM >= 0x070f04 /* 7.15.4 */
```

Comment:
According to the earlier header definition, these functions were conditionally declared